futurex_openedx_extensions/helpers/tasks_utils.py

```diff
@@ -40,19 +42,17 @@ def _get_view_class_instance(path: str) -> Any:
     )
 
 
-def _get_mocked_request(url: str, user: Any, fx_info: dict, query_params: dict) -> Request:
+def _get_mocked_request(url_with_query_str: str, fx_info: dict) -> Request:
     """Create mocked request"""
     factory = APIRequestFactory()
-    mocked_request = factory.get(url)
-    mocked_request.user = user
+    mocked_request = factory.get(url_with_query_str)
+    mocked_request.user = fx_info.get('user')
```

Comment:
This must fail if not filled
```suggestion
    mocked_request.user = fx_info['user']
```